docs/intro/reusable-apps.txt

```diff
@@ -198,91 +198,61 @@ this. For a small app like polls, this process isn't too difficult.
    license. Just be aware that your licensing choice will affect who is able
    to use your code.
 
-#. Next we'll create ``pyproject.toml``, ``setup.cfg``, and ``setup.py`` files
-   which detail how to build and install the app. A full explanation of these
-   files is beyond the scope of this tutorial, but the `setuptools
-   documentation <https://setuptools.pypa.io/en/latest/>`_ has a good
-   explanation. Create the ``django-polls/pyproject.toml``,
-   ``django-polls/setup.cfg``, and ``django-polls/setup.py`` files with the
-   following contents:
+#. Next we'll create the ``pyproject.toml`` file which details how to build and
+   install the app. A full explanation of this file is beyond the scope of this
+   tutorial, but the `Python Packaging User Guide
+   <https://packaging.python.org/en/latest/>`_ has a good explanation. Create
```

Comment:
After visiting this link, the first thing I had to do was put pyproject.toml in the search box. I think it would be nice to link to a more specific page like https://packaging.python.org/en/latest/guides/writing-pyproject-toml/.